sql-db/hibernate-fulltext-search/src/main/java/io/quarkus/ts/hibernate/search/VegetableResource.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.ts.hibernate.search;
+
+import java.util.List;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Response;
+
+import org.hibernate.search.mapper.orm.session.SearchSession;
+
+import io.smallrye.common.annotation.Blocking;
+
+@ApplicationScoped
+@Produces("application/json")
+@Consumes("application/json")
+@Path("/{tenant}/vegetables")
```

Comment:
Why do we use this variable in the path? I do not see it being used anywhere

Reply:
> Why do we use this variable in the path? I do not see it being used anywhere

it is used by Hibernate tenant resolver, this modules also uses tenants